src/mqtt/MQTT.cpp

```diff
@@ -379,13 +379,13 @@ void MQTT::sendSubscriptions()
             std::string topic = cryptTopic + channels.getGlobalId(i) + "/+";
             LOG_INFO("Subscribing to %s\n", topic.c_str());
             pubSub.subscribe(topic.c_str(), 1); // FIXME, is QOS 1 right?
-#ifndef ARCH_NRF52                              // JSON is not supported on nRF52, see issue #2804
+// #ifndef ARCH_NRF52                              // JSON is not supported on nRF52, see issue #2804 ### Fixed by using ArduinoJSON ###
```

Comment:
We can’t just remove this, because it will still crash on nRF52 boards if you don’t use ArduinoJson (people use MQTT with client proxy over BLE as well). Please change to `#if !defined(ARCH_NRF52) || defined(NRF52_USE_JSON)`.

Reply:
Hi @GUVWAF 
Got it. Will change it.